status-go-version.json

```diff
@@ -3,7 +3,7 @@
     "_comment": "Instead use: scripts/update-status-go.sh <rev>",
     "owner": "status-im",
     "repo": "status-go",
-    "version": "v0.117.1a",
-    "commit-sha1": "b4bdfd3df6cf5fb91ab2d0e9f3b38e8d1b9703e5",
-    "src-sha256": "1s50q53p1j5ysnbj88c52zb2rh1ms21pnf6bjc7wja8gzcpih2wl"
+    "version": "14560-support-compressed-keys-for-communities",
```

Comment:
status-go version is my branch because that PR can not be merged in due to some CI issues.
I will update this to point to develop once that is fixed.
This is just to ensure that builds work fine and QA can test the fix.

Reply:
updated status-go version to sha1 of latest commit in status-go